tests/Integration/Controllers/AdminSearchControllerCoreTest.php

```diff
@@ -31,10 +31,15 @@
 use AdminSearchControllerCore;
 use Context;
 use Employee;
-use PHPUnit\Framework\TestCase;
+use Symfony\Bundle\FrameworkBundle\Test\KernelTestCase;
 
-class AdminSearchControllerCoreTest extends TestCase
+class AdminSearchControllerCoreTest extends KernelTestCase
 {
+    public function setUp(): void
+    {
+        self::bootKernel();
+    }
```

Comment:
Can someone explain this adding ?

Reply:
You're right seems like it works without it, I don't remember why I added it (removing it)

Reply:
👍